### ISSUE TITLE
COM port "file not found" on windows

### ISSUE THREAD

**Issue by kubaraczkowski** [panStamp/arduino_nrg#1]
Not sure if it's only me, but when programming NRG1.0 board I needed to change the msp430-gdb command to do `"target remote \\.\COM26"` on windows instead of simply `"target remote COM26"` which is default.
Could somebody perhaps try it as well? (tested on win7 64b)

**Comment by dberenguer** [comment]
This is probably an old bug. We didn't fully test NRG 1.0 on the newest (at
that time) Windows distributions.

Daniel Berenguer
http://www.panstamp.com

**Comment by kubaraczkowski** [comment]
Yes, that's what I thought. I just have a bunch of them still and wanted to
make some use of them.

**Comment by dberenguer** [comment]
Does your fix work?

Daniel Berenguer
http://www.panstamp.com

**Comment by kubaraczkowski** [comment]
Well, for the moment I've only tried by taking the error message,
extracting the msp430-gdb command which does the programming, adapting the
COM port notation, running - gets the work done.
Did not try adapting in the arduino software...
